app/assets/src/stylesheets/02-molecules/project-summary.scss

```diff
@@ -8,7 +8,7 @@
 // assumes `current_user` is set; requires local `project`
 // -->
 // <div class="project-summary">
-//   <h2>Space Unicorns</h2>
+//   <h3>Space Unicorns</h3>
```

Comment:
I demoted the summary to an H3, as it's likely we'll want to use H2s to group summaries by type (in this particular case, I grouped them under Pending and Approved).

Reply:
Let's make "first heading level" a parameter that can be passed in, then -- we have existing pages that don't have h2s heading up the project list(s), and I'd like us to be strict about our page outlining for SEO and a11y reasons.

This one is blocking unless you have a strong reason it shouldn't be -- I would like us to be pretty strict about a11y on this project.

Reply:
Sounds reasonable